ngraph/core/include/ngraph/op/constant.hpp

```diff
@@ -285,6 +285,12 @@ namespace ngraph
                 ///        Repeated values are allowed.
                 AxisSet get_axis_set_val() const;
 
+                /// \brief Update Constant shape. New shape size must equal to the data elements
+                /// count
+                ///
+                /// \param shape The shape of the tensor constant.
+                void set_data_shape(const Shape& shape);
```

Comment:
Why do we need this method? 
Maybe the better name is reshape? because we don't change the size of constant

Reply:
Yes and `set_data_shape` doesn't change data size. When we create constant we put shape and data to the Constant ctor. So for me this method name absolutely transparent and doesn't mean change of data size.